test/e2e/specs/editor/various/new-post-default-content.spec.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'new editor filtered state', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activatePlugin(
+			'gutenberg-test-plugin-default-post-content'
+		);
+	} );
+
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await requestUtils.deactivatePlugin(
+			'gutenberg-test-plugin-default-post-content'
+		);
+	} );
+
+	test( 'should respect default content', async ( { editor, page } ) => {
+		const content = await editor.getEditedPostContent();
+
+		await page.click( 'role=button[name="Excerpt"i]' );
+
+		// Assert they match what the plugin set.
+		await expect(
+			page.locator( 'role=textbox[name="Add title"i]' )
+		).toHaveText( 'My default title' );
+		expect( content ).toBe( 'My default content' );
```

Comment:
```suggestion
		await expect
			.poll( editor.getEditedPostContent )
			.toBe( 'My default content' );
```

Suggestion: I would recommend getting content right before the assertion. In this case, you can even inline it.